disperser/batcher/encoding_streamer.go

```diff
@@ -120,9 +121,11 @@ func (e *EncodingStreamer) Start(ctx context.Context) error {
 			case response := <-encoderChan:
 				err := e.ProcessEncodedBlobs(ctx, response)
 				if err != nil {
-					if !errors.Is(err, context.Canceled) {
-						e.logger.Error("error processing encoded blobs", "err", err)
+					if strings.Contains(err.Error(), context.Canceled.Error()) {
```

Comment:
`errors.Is(err, context.Canceled)`?

Reply:
That didn't work as context canceled message in `err` is not wrapped as `error` but as rpc status.
This is a workaround.